dss-document/src/main/java/eu/europa/esig/dss/validation/DiagnosticDataBuilder.java

```diff
@@ -806,7 +806,7 @@ private XmlCertificate getXmlCertificate(Set<DigestAlgorithm> usedDigestAlgorith
 		final XmlCertificate xmlCert = new XmlCertificate();
 
 		xmlCert.setId(certToken.getDSSIdAsString());
-                xmlCert.setBase64Encoded(Utils.toBase64(certToken.getEncoded()).getBytes());
```

Comment:
Not b64-coding of the result from certToken.getEncoded() leads to an exception during unmarshalling of DiagnosticData in a remote validation of signatures because DiagnosticData.xsd declares the X059Certificate as: 
<xs:element name="Base64Encoded" type="xs:base64Binary" minOccurs="0" />
See at: 
https://github.com/esig/dss/commit/2d5cb71c1e63a9d1aced984f9f51963f8156c75b

On the other hand certToken.getEncoded() delivers X.509 certificates  encoded as ASN.1 DER
and not Base64-coded data.